src/App/Pages/Vault/GroupingsPage/GroupingsPageViewModel.cs

```diff
@@ -424,8 +472,23 @@ public async Task SyncAsync()
 
         private async Task LoadDataAsync()
         {
+            string orgId = null;
+            var decCiphers = await _cipherService.GetAllDecryptedAsync();
+            if (IsVaultFilterMyVault)
+            {
+                _allCiphers = decCiphers.Where(c => c.OrganizationId == null).ToList();
+            }
+            else if (IsVaultFilterOrgVault)
+            {
+                orgId = await GetVaultFilterOrgIdAsync();
+                _allCiphers = decCiphers.Where(c => c.OrganizationId == orgId).ToList();
+            }
+            else
+            {
+                _allCiphers = decCiphers;
+            }
```

Comment:
Can we extract it to another method? `LoadDataAsync` is IMO already too large, it could be something like:
`async Task<string> FillAllCiphersAndGetOrgIdIfNeededAsync();`